src/main/swagger/problem/v1/problem-v1.yaml

```diff
@@ -36,7 +40,17 @@ definitions:
       instance:
         type: string
         format: uri
-        description: A URI reference that identifies the specific occurrence of the problem. It may or may not yield further information if dereferenced.
+        description: An absolute URI reference that identifies the specific occurrence of the problem. It may or may not yield further information if dereferenced.
+    example:
+      {
+        "type": "urn:problem-type:belgif:payloadTooLarge",
+        "href": "https://www.gcloud.belgium.be/rest/problems/payloadTooLarge", # location might change in the future
```

Comment:
add .html  extension => https://www.gcloud.belgium.be/rest/problems/payloadTooLarge.html for link to work